lib/src/map_util.dart

```diff
@@ -1,23 +1,14 @@
 import 'objects.dart';
 
+//TODO why is this in a class with a Util suffix when that is not the convention in the rest of the library
```

Comment:
This does match our convention:

https://github.com/jesims/docs/blob/master/Coding-Conventions/Dart.md#utilities

Reply:
so I think we can remove the todo

Reply:
Oh, I don't remember being notified of that update to our Dart convention docs. No worries, I'll remove the TODO

Reply:
You were in our entire dev discussion about it when we made the decision :) Every dev agreed